composer.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "netlogix/nlxproductcomparison",
+    "description": "This plugin customize shopware product comparison logic",
+    "type": "shopware-plugin",
+    "license": "proprietary",
+    "authors": [
+        {
+            "name": "netlogix GmbH & Co. KG",
+            "email": "websolutions@netlogix.de"
+        }
+    ],
+    "extra": {
+        "installer-name": "nlxProductComparison"
+    },
+    "autoload": {
+        "psr-4": {
+            "nlxProductComparison\\": "."
+        }
+    },
+    "require": {
+        "php": "^7.2"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^5.7",
+        "shopware/shopware": "^5.6",
+        "solutiondrive/sd-test-environment-shopware": "~0.2",
+        "solutiondrive/standalone-build-tools": "^1.0"
+    },
+    "config": {
+        "sort-packages": true,
+        "standalone-build-tools": {
```

Comment:
Here we should use the correct php versions as well

Reply:
What do you mean with the correct php version?

Reply:
See the other comment above in the .travis.yml "We should directly test php 7.3 and php 7.4 as well"

Reply:
Done https://github.com/netlogix/nlxProductComparison/pull/1/commits/c4aeb2bb669e0429e33fc5d13cc7af0f15954d88